README.md

```diff
@@ -307,13 +227,52 @@ orchestrator.register_agent(attractions_agent)
 # Query the orchestrator.
 runner = orchestrator.ask("Create a two-day itinerary for a trip to Rome. Please don't ask me any questions; just provide the best itinerary you can.")
 print(runner.output)
+
 ```
+## Community
+
+Have questions or feedback? Reach out via:
+- [GitHub Issues](https://github.com/FalkorDB/GraphRAG-SDK/issues)
+- Join our [Discord](https://discord.com/invite/6M4QwDXn2w)
+
+⭐️ If you find this repository helpful, please consider giving it a star!
+
+
+
+## FAQ
+**Which databases are supported?**
+
+GraphRAG-SDK is optimized for FalkorDB. Other backends may require adapters.
+
+**How scalable is the SDK?**
+
+GraphRAG-SDK is designed for multi-tenancy and large-scale applications. Performance depends on FalkorDB deployment configuration.
+
+**What is GraphRAG?**

```

Comment:
Is it a bit too late for this question?
if you want to keep it I suggest you bump it to the very top.

Reply:
removed